src/type/__tests__/introspection-test.js

```diff
@@ -63,6 +63,7 @@ describe('Introspection', () => {
               interfaces: [],
               enumValues: null,
               possibleTypes: null,
+              specifiedByUrl: null,
```

Comment:
nit: Better to keep the same order of fields as they specified in introspection query, so `specifiedBy` should be after `name`.

Reply:
Fixed in d6e4f37a61f0f4597a008280822cec3a13e11794